blockbase/theme.json

```diff
@@ -116,6 +116,38 @@
 				"secondary": "var(--wp--preset--color--secondary)",
 				"selection": "var(--wp--preset--color--selection)"
 			},
+			"colorPalettes": {
+				"palette-1": {
+					"label": "Featured",
```

Comment:
I think there is a mechanism now for labels for presets to get translated, but is there a way yet to mark that a custom variable needs to be translated?

Reply:
I think we do it like [this](https://github.com/WordPress/gutenberg/pull/29828). Since we haven't done this for any of the strings that we have (such as the template parts) I think this merits its own separate PR